lib/dialects/sqlite/query-generator.js

```diff
@@ -268,6 +340,29 @@ const QueryGenerator = {
       + `DROP TABLE ${quotedBackupTableName};`;
   },
 
+  alterConstraintQuery(tableName, attributes, createTableSql) {
```

Comment:
This is a private method, right. I don think user will ever call this as it require `createTableSql`. It will be good idea to prefix such private methods with `_`